pkg/services/accesscontrol/acimpl/accesscontrol.go

```diff
@@ -117,8 +119,24 @@ func (a *AccessControl) evaluateZanzana(ctx context.Context, user identity.Reque
 	}
 
 	return eval.EvaluateCustom(func(action, scope string) (bool, error) {
-		// FIXME: Implement using new schema / apis
-		return false, nil
+		kind, _, identifier := accesscontrol.SplitScope(scope)
+		namespace := claims.OrgNamespaceFormatter(user.GetOrgID())
```

Comment:
This is probably fine for now but if we are running this as a stack the namespace should be based on the stack id.

Can you leave a fixme comment for that?